kubernetes/pods.go

```diff
@@ -43,7 +43,9 @@ func (a *Adapter) PodInformer(ctx context.Context, endpointChan chan<- []string)
 		time.Sleep(resyncInterval)
 	}
 	for _, pod := range podList {
-		podEndpoints.Store(pod.Name, pod.Status.PodIP)
+		if isPodRunning(pod) {
+			podEndpoints.Store(pod.Name, pod.Status.PodIP)
+		}
```

Comment:
> Add a test for it maybe?
>> NP, covered by TestAdapter_PodInformer and explicitly now

Well, I meant the listing part - the filtering was introduced but no test was changed which means it was not covered.

Reply:
added - the test should contain all conditions now